terraform/api.tf

```diff
@@ -1,11 +1,27 @@
 resource "aws_sqs_queue" "bundle_events" {
   name = "${var.BUNDLE_EVENTS_QUEUE_NAME}"
   visibility_timeout_seconds = 900
+  redrive_policy = jsonencode({
+    "deadLetterTargetArn" = "${aws_sqs_queue.bundle_events_dlq.arn}",
+    "maxReceiveCount" = 5
+  })
+}
+
+resource "aws_sqs_queue" "bundle_events_dlq" {
```

Comment:
do we need separate dead letter queues? Can we just have one that all of the queues use?

Reply:
Good point, we should probably just use one.

Reply:
Reconfigured TF code to use just one SQS queue.